lib/src/lifecycle_module.dart

```diff
@@ -220,8 +220,8 @@ abstract class LifecycleModule extends SimpleModule with Disposable {
   @protected
   void specifyStartupTiming(
     StartupTimingType specifier, {
-    Map<String, dynamic> tags = const {},
-    List<Reference> references = const [],
+    Map<String, dynamic>? tags = const {},
+    List<Reference>? references = const [],
```

Comment:
I think these can be removed as well?
```suggestion
    Map<String, dynamic> tags = const {},
    List<Reference> references = const [],
```

Reply:
and some of the `!` below can be removed it accepting this suggestion.

Reply:
Removed `?`. Got unnecessary `!` , so removed the same